roles/docker/tasks/main.yml

```diff
@@ -36,6 +36,15 @@
   tags:
     - prebake-for-dev
 
+- stat: path=/var/docker_images
+  register: docker_images
```

Comment:
I get the feeling this tag has too much power and needs to be factored out into something cleaner. the fact we have to use it everywhere smells.

that said I will make the change, let's just look soon into a different way of doing this in the future.